kommunicate/src/main/java/com/applozic/mobicomkit/api/conversation/MobiComConversationService.java

```diff
@@ -345,7 +347,7 @@ public synchronized List<Message> getMessagesForParticularThread(Long startTime,
     }
 
 
-    public synchronized List<Message> getMessages(Long startTime, Long endTime, Contact contact, Channel channel, Integer conversationId, boolean isSkipRead, boolean isForSearch) {
+    public synchronized List<Message> getMessages(Boolean isAgentApp, Long startTime, Long endTime, Contact contact, Channel channel, Integer conversationId, boolean isSkipRead, boolean isForSearch) {
```

Comment:
Check if we need to pass it as an argument or find some better way to identify if it is an agent app. Or changes it to something generic like, show summaryView or customView

Reply:
Done, Added in [2410150](https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/pull/565/commits/2410150a3bea2d2b2d2dfd0479b5012cc515f3d8)